components/network/openldap/files/ldap-olslapd

```diff
@@ -31,7 +33,7 @@ typeset -r VARRUNDIR=/var/openldap/run
 typeset -r VARDATADIR=/var/openldap/openldap-data
 typeset -r PIDFILE=${VARRUNDIR}/slapd.pid
 typeset -r CONF_FILE=/etc/openldap/slapd.conf
-typeset -r SLAPD="/usr/lib/slapd -u ${LDAPUSR} -g ${LDAPGRP} -f ${CONF_FILE}"
+typeset -r SLAPD="/usr/lib/amd64/slapd -u ${LDAPUSR} -g ${LDAPGRP} -f ${CONF_FILE}"
```

Comment:
1) Don't use amd64 in scripts (we have /usr/lib/64)
2) If we always use 64-bit slapd, what's the purpose of delivering 32-bit one? (PREFERRED_BITS= 64 ?)

Reply:
Not going with this this time.